packages/xstate-inspect/src/browser.ts

```diff
@@ -86,6 +87,8 @@ const getFinalOptions = (options?: Partial<InspectorOptions>) => {
   };
 };
 
+const patchedInterpreters = new Set<AnyInterpreter>();
```

Comment:
I'd like to revisit the `@xstate/inspect` API in the future but it's worth fixing this asap, especially given that React 18 restarts services in StrictMode by default.